src/gt4sd/configuration.py

```diff
@@ -53,13 +54,29 @@ class GT4SDConfiguration(BaseSettings):
     gt4sd_s3_access_key: str = "6e9891531d724da89997575a65f4592e"
     gt4sd_s3_secret_key: str = "5997d63c4002cc04e13c03dc0c2db9dae751293dab106ac5"
     gt4sd_s3_secure: bool = True
-    gt4sd_s3_bucket: str = "gt4sd-cos-algorithms-artifacts"
+    gt4sd_s3_bucket_algorithms: str = "gt4sd-cos-algorithms-artifacts"
+    gt4sd_s3_bucket_properties: str = "gt4sd-cos-properties-artifacts"
 
     gt4sd_s3_host_hub: str = "s3.par01.cloud-object-storage.appdomain.cloud"
     gt4sd_s3_access_key_hub: str = "d9536662ebcf462f937efb9f58012830"
     gt4sd_s3_secret_key_hub: str = "934d1f3afdaea55ac586f6c2f729ac2ba2694bb8e975ee0b"
     gt4sd_s3_secure_hub: bool = True
-    gt4sd_s3_bucket_hub: str = "gt4sd-cos-hub-algorithms-artifacts"
+    gt4sd_s3_bucket_hub_algorithms: str = "gt4sd-cos-hub-algorithms-artifacts"
+    gt4sd_s3_bucket_hub_properties: str = "gt4sd-cos-hub-properties-artifacts"
+
+    gt4sd_s3_modules: List[str] = ["algorithms", "properties"]
+    get_local_cache_path: Dict[str, str] = {
```

Comment:
I would remove the get prefix from this attributes, they can be misleading and suggesting that those are methods